.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Feature Request
+about: Request a new feature
+labels: "enhancement"
+
+--
+<!--
+Please first search existing issues for the feature you are requesting; it may already exist, even as a closed issue.
```

Comment:
> even as a closed issue

I'd rather not mention the closed issues. I don't want to encourage comments on closed issues.

Mentioning to search first for their issue is good, though 👍